README.md

```diff
@@ -64,6 +64,7 @@ For details on KairosDB, please refer to [documentation](https://kairosdb.github
 Task manifest configuration is described in [snap's documentation](https://github.com/intelsdi-x/snap/blob/master/docs/TASKS.md). In order to use KairosDB publisher you have to add section "publish" then specify following options:
 - `"host"` - KairosDB host address (ex. `"127.0.0.1"`)
 - `"port"` -  KairosDB REST API port (ex. `"8083"`)
+- `"useDynamic"` -  Use dynamic namespaces for metric and tags (ex. `true`)
```

Comment:
You right. Thanks
https://github.com/julcollas/snap-plugin-publisher-kairosdb/commit/0ef26626f68119ffc2467200b2be7da7cb923e87